bin.src/make_latiss_export.py

```diff
@@ -52,40 +63,66 @@ def main():
     args = _make_parser().parse_args()
     src_butler = daf_butler.Butler(args.src_repo)
 
+    if args.target_repo:
+        target_butler = daf_butler.Butler(args.target_repo, writeable=False)
+    else:
+        # If no target_butler ig given, create an empty one.
+        temp_dir = tempfile.TemporaryDirectory()
+        config = daf_butler.Butler.makeRepo(temp_dir.name)
+        target_butler = daf_butler.Butler(config)
+
     with time_this(msg="Exporting datasets and collections", level=logging.INFO):
-        _export_for_copy(src_butler)
+        _export_for_copy(src_butler, target_butler)
+
+    if not args.target_repo:
+        temp_dir.cleanup()
```

Comment:
I guess a context manager is impractical here, so please put this in a `finally` block.

Reply:
I couldn't come up with a way that makes the `finally` block looks better than the context manager, so I'm going with the context manager.